js/Repeater/RepeaterPreviewService.js

```diff
@@ -97,7 +97,14 @@ class RepeaterPreviewService {
                     true;
             })
             .forEach(preview => {
-                $(preview).find(this.queryService.getQuery('preview-ui')).toggleClass('disabled');
+                let $previewUi = $(preview).find(this.queryService.getQuery('preview-ui'));
+                $previewUi.toggleClass('disabled');
+
+                if ($previewUi.attr('disabled')) {
```

Comment:
Was just reading up on whether this is the best way to check, jq recommend we use `prop()` here 

> As of jQuery 1.6, the .attr() method returns undefined for attributes that have not been set. To retrieve and change DOM properties such as the checked, selected, or disabled state of form elements, use the .prop() method.

so maybe a more robust solution might be `if ($previewUi.prop('disabled') !== undefined) {`